testnet/m1_facilitator/m1_facilitator/fund_facilitator/Fund_Avatar.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import axios from 'axios';
+import BigNumber from 'bignumber.js';
+import Web3 from 'web3';
+import Command from '../../../../src/m1_facilitator/commands/Command';
+import Manifest from '../../../../src/m1_facilitator/manifest/Manifest';
+
+// Threshold in ether below which avatar account will be funded from faucet.
+const AVATAR_ACCOUNT_THRESHOLD = new BigNumber(50);
+
+const FAUCET_URL = 'https://faucet.mosaicdao.org';
+
+enum Chains {
+  Goerli = '5',
+  Hadapsar = '1405',
+}
+
+/**
+ * Returns balance of avatar account.
+ */
+async function checkBalance(account: string, auxWeb3: Web3): Promise<BigNumber> {
+  const auxAvatarBalanceWei = await auxWeb3.eth.getBalance(account);
```

Comment:
just use `accountBalance` instead of `auxAvatarBalanceWei`
Do not convert to Ether, compare in atto/wei

Reply:
Yes, changed.

Reply:
`auxAvatarBalanceWei ` Changed to `accountBalance`. Comparing in wei.